src/main/resources/application.yml

```diff
@@ -57,3 +57,11 @@ management:
     readinessstate:
       enabled: true
 debug: false
+application:
+  minio-remote:
+    endpoint: ${AWS_URL:http://127.0.0.1:9000/}
+    region: ${AWS_REGION:}
+    bucket: ${AWS_BUCKET:}
+    accessKey: ${AWS_ACCESS_KEY_ID:}
+    secretKey: ${AWS_SECRET_ACCESS_KEY:}
+    composeWithAwsSdk: ${LOCAL_FS_COMPOSE_WITH_AWS_SDK:false}
```

Comment:
composeWithAwsSdk -> isAwsSdk because we support now not only append (compose) but also write method